src/services/profile.service.js

```diff
@@ -0,0 +1,38 @@
+import models from '../database/models';
+
+const { user } = models;
+
+/**
+ * @description class ProfileService handles everything regarding to profile
+ */
+export default class ProfileService {
+  /**
+   * @param{number} requestedProfile
+   * @returns{object} user
+   */
+  static readProfile = async (requestedProfile) => {
+    const userData = await user.findOne({ where: { id: requestedProfile } });
+    return userData;
+  }
+
+  /**
+   * @param{object} profileData
+   * @param{string} email
+   * @returns{object} newProfileData
+   * @description update profile and returns new updated profile data
+   */
+  static updateProfile = async (profileData, email) => {
+    const updatedUser = await user.update(profileData, { where: { email }, returning: true });
+    return updatedUser;
+  }
+  
+  /**
+   * @param{string} newPassword
+   * @param{string} email
+   * @returns{string} new password
+   */
+  static changePassword = async (newPassword, email) => {
+    const isPasswordUpdated = await user.update({ password: newPassword }, { where: { email } });
+    console.log(isPasswordUpdated);
```

Comment:
Do you really need to `console.log(isPasswordUpdated);`

Reply:
Animida, that console.log is not needed in deployment codes, I remove it